migrations/898-comm-cc-nonuser.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE `comm_thread_cc`
+    MODIFY COLUMN `user_id` int(11) unsigned NULL,
+    ADD COLUMN `nonuser_email` varchar(255) NULL;
```

Comment:
I might even just manually email without using the CC model if the Mozilla contact isn't found. That might be a leaner solution than upkeeping a list of Moz contacts.